src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -48,8 +48,8 @@
         (wds/rollup :mean avg-name [:gss-code :sex :age]))))
 
 ;;Method 3: Calculate trend (perform linear regression) over range of histrical data &
```

Comment:
I'm going to change the assoc figure to match this order - will send you file in slack